docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -1,18 +1,11 @@
 [id="deploying-backend-services_{context}"]
 
-= Deploying backend services
+= Deploying back-end services
 
-Create the `OpenStackControlPlane` custom resource (CR) with basic backend services deployed, and all the {rhos_prev_long} ({OpenStackShort}) services disabled. This will be the foundation of the control plane.
-
-In subsequent steps, you import the original databases and then add
-{OpenStackShort} control plane services.
+Create the `OpenStackControlPlane` custom resource (CR) with the basic back-end services deployed, and disable all the {rhos_prev_long} ({OpenStackShort}) services. This CR is the foundation of the control plane.
 
 .Prerequisites
 
-* The cloud that you want to adopt is up and running, and it is on the
-{OpenStackShort} {rhos_prev_ver} release.
-* All control plane and data plane hosts of the source cloud are up
-and running, and continue to run throughout the adoption procedure.
 * The `openstack-operator` is deployed, but `OpenStackControlPlane` is
```

Comment:
I removed the first two prerequisites because they seemed more general to adoption rather than specific to this procedure. I will likely move those general prereqs to an “adoption preparation checklist”, which I’m developing.

Is the prereq, “The `openstack-operator` is deployed, but `OpenStackControlPlane` is
not deployed," more generic or specific to deploying back-end services?

Reply:
I think we should actually keep these, they were an explicit request based on past experience. Folks were sometimes starting the upgrade procedure when some nodes were broken/offline but not properly removed from the OpenStack deployment, and it caused a failure and escalation.